docs-v2/content/development/setting-up-a-dev-environment.md

```diff
@@ -0,0 +1,106 @@
+# Setting up a dev environment (React)
```

Comment:
Why React only?

Reply:
It shouldn't be I think.

Reply:
I'll change it to be setting up a dev environment, then a section for each different platform (React, ReactNative, Angular, etc)